src/containers/AskGlific/AskGlific.module.css

```
@@ -0,0 +1,530 @@
+  cursor: pointer;
+  font-size: 0.9rem;
+  color: #333;
+  min-width: 160px;
+}
+
+.DisplayModeItem:hover {
+  background-color: #f5f5f5;
+}
+
+.DisplayModeItemActive {
+  composes: DisplayModeItem;
```

Stylelint (17.10.0)

[error] Expected "DisplayModeItem" to be "displaymodeitem" (value-keyword-case)

(value-keyword-case)